### ISSUE TITLE
Build Failed

### ISSUE THREAD

**Issue by DharaniRenu1** [headuck/react-native-zebra-rfid#1]
![screenshot from 2019-01-20 15-48-03](https://user-images.githubusercontent.com/37014905/51437931-3da9fc80-1ccb-11e9-8969-35bcf8e86b07.png)

**Comment by headuck** [comment]
Please check the minsdk level of your project as described in the error message.

**Comment by DharaniRenu1** [comment]
Yeah, I ran the command. How to scan the id. Please guide me to get my RFID.

**Comment by DharaniRenu1** [comment]
I couldn't clone and your's repository too.

**Comment by johainnessalise** [comment]
How do you add the antenna power settings?
